cluster-autoscaler/cloudprovider/oci/README.md

```diff
@@ -49,6 +49,17 @@ Allow dynamic-group acme-oci-cluster-autoscaler-dyn-grp to use vnics in compartm
 Allow dynamic-group acme-oci-cluster-autoscaler-dyn-grp to inspect compartments in compartment <compartment-name>
 ```
 
+### If using Workload Identity
```

Comment:
Could you put a [link](https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contenggrantingworkloadaccesstoresources.htm) here as well as a note that this provider is only available for [Oracle Container Engine for Kubernetes](https://www.oracle.com/cloud-native/container-engine-kubernetes/) (i.e. node-pools, not _instance-pools_). (assuming that is the case)

Reply:
As noted by Jesse, one major distinction that should be noted is that this should work for any node (whether allocated via a node pool or instance pool), as long as the underlying Kubernetes Masters are based on OKE, and not a 3rd party Kubernetes Control Plane (e.g. minikube).

Reply:
Updated README, and splitted the PR into 2 commits SDK update and Workload Identity implementation for easier cherry picking to other branches.